CODING.md

```diff
@@ -64,6 +64,20 @@ Note: Chrome in headless mode doesn't support the WebGL `EXT_frag_depth`
 extension. So rendering may differ and some bugs can only be present in headless
 mode.
 
+### Useful commands for continuous testing
+
+If you wish to have unit tests to continuously run, you can append to your
+running command `-- --watch`. You will be getting in this case `npm run
+test-unit -- --watch`.
+
+If you want to work on a single test and debug it, without having all the extra
+output, you can use this command:
+```sh
+./node_modules/.bin/cross-env BABEL_DISABLE_CACHE=1 \
```

Comment:
How about `npx cross-env` rather than `./node_modules/.bin/cross-env` (and similarly for `mocha`)

Reply:
Oh I finally understand what `npx` is, thanks :smile: Will do